src/software/pipeline/main_cameraInit.cpp

```diff
@@ -335,6 +354,27 @@ int aliceVision_main(int argc, char **argv)
       return EXIT_FAILURE;
   }
 
+  std::list<std::string> colorProfileList;
+  if (!colorProfileDatabaseDirPath.empty() && !fs::is_directory(colorProfileDatabaseDirPath))
+  {
+      ALICEVISION_LOG_ERROR("The specified database for color profiles does not exist.");
+      return EXIT_FAILURE;
+  }
+  else if (!colorProfileDatabaseDirPath.empty())
+  {
+      fs::path targetDir(colorProfileDatabaseDirPath);
+      fs::directory_iterator it(targetDir), eod;
+
+      BOOST_FOREACH(fs::path const& p, std::make_pair(it, eod))
```

Comment:
I think you can use c++11 this way
```suggestion
      for(const auto& p,  boost::make_iterator_range(directory_iterator(targetDir), {}))
```
see https://stackoverflow.com/a/20925615